scripts/docker/build_docker.py

```diff
@@ -488,16 +527,16 @@ def local_image(self) -> str:
         return f"{self.name}:{self.tag}"
 
     @property
-    def remote_docker_repos(self) -> Tuple[str, ...]:
+    def remote_docker_repos(self) -> Dict[str, ContainerRegistry]:
         return self.project_builder.remote_docker_repos
 
     @property
-    def remote_images(self) -> Tuple[str, ...]:
+    def remote_images(self):
         remote_tags = (self.tag, ProjectBuilder.latest_tag) if self.project_builder.project_arguments.update_latest \
             else (self.tag,)
         return tuple(
-            f"{repo}/{self.name}:{remote_tag}"
-            for repo in self.remote_docker_repos
+            (name, f"{name}/{self.name}:{remote_tag}")
+            for name, registry in self.remote_docker_repos.items()
```

Comment:
Just since there are a few `name`s running around, I'd suggest being more explicit:
```suggestion
            (registry_name, f"{registry_name}/{self.name}:{remote_tag}")
            for registry_name, registry in self.remote_docker_repos.items()
```

Reply:
Agree, I refactored the var name as you suggested.